fvm/environment/account_info.go

```diff
@@ -254,3 +269,21 @@ func (info *accountInfo) GetAccount(
 
 	return account, nil
 }
+
+func (info *accountInfo) GetAccountKeys(
+	address flow.Address,
+) (
+	[]flow.AccountPublicKey,
+	error,
+) {
+	defer info.tracer.StartChildSpan(trace.FVMEnvGetAccount).End()
+
+	//accountKeys, err := info.accounts.GetPublicKey(address, 0)
+
+	account, err := info.accounts.Get(address)
```

Comment:
maybe we can just add this to the `Accounts` interface:
https://github.com/onflow/flow-go/blob/091b7412aa23629ecca51255958d4acb94d04183/fvm/environment/accounts.go#L251